components/sidebar/add_channel_dropdown.tsx

```diff
@@ -189,14 +189,12 @@ const AddChannelDropdown = ({
                 placement='top'
                 overlay={tooltip}
             >
-                <>
-                    <button
-                        className={'AddChannelDropdown_dropdownButton'}
-                        aria-label={intl.formatMessage({id: 'sidebar_left.add_channel_dropdown.dropdownAriaLabel', defaultMessage: 'Add Channel Dropdown'})}
-                    >
-                        <i className='icon-plus'/>
-                    </button>
-                </>
+                <button
+                    className={'AddChannelDropdown_dropdownButton'}
+                    aria-label={intl.formatMessage({id: 'sidebar_left.add_channel_dropdown.dropdownAriaLabel', defaultMessage: 'Add Channel Dropdown'})}
+                >
+                    <i className='icon-plus'/>
```

Comment:
@pvev I know all the reviews are done, but can we use compass icons here

Reply:
sure, but do you mean these? https://mattermost.github.io/compass-icons/

Reply:
Yeah these.